spec.go

```diff
@@ -53,6 +53,12 @@ const (
 	starBit = 1 << 63
 )
 
+// isOneOff returns a true or false if this schedule should only be ran once.
+// For ConstantDelaySchedule this will ALWAYS return false
```

Comment:
Is this comment right? This is not a **ConstantDelaySchedule** is it?

Reply:
Yeah bad commenting. Fixed: https://github.com/NeilGerring/cron/pull/1/commits/3c74ada4d9af62ef6b3a047223a4137c3c894a0b